metapool/tests/test_igm.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+import tempfile
+import sys
+from datetime import datetime
+
+from metapool.igm import IGMManifest
+from openpyxl import load_workbook, Workbook
+
+
+class IGMManifestTests(unittest.TestCase):
+    def test_raises_error_with_missing_properties(self):
+        m = IGMManifest()
+        with self.assertRaisesRegex(ValueError, 'project_number cannot be '
+                                    'empty, you need to set a value'):
+            m.write()
+
+    def test_raises_error_with_missing_properties_single(self):
+        m = IGMManifest()
+        m.project_number = 1000
+        m.task_number = 3
+        m.number_of_lanes = 1
+        m.pools = ['TMI_AGP_300_302_304_308']
+        with self.assertRaisesRegex(ValueError, 'number_of_samples cannot be '
+                                    'empty, you need to set a value'):
+            m.write()
+
+    def test_default_values(self):
+        m = IGMManifest()
+
+        self.assertEqual(m.submission_date,
+                         datetime.strftime(datetime.today(), '%m/%d/%y'))
+        self.assertEqual(m.institute, 'Knight Lab')
+        self.assertEqual(m.pi_name, 'Dr. Knight')
+        self.assertEqual(m.pi_email, 'mackenzie.m.bryant@gmail.com')
+        self.assertEqual(m.contact_name, 'MacKenzie Bryant')
+        self.assertEqual(m.contact_email, 'mackenzie.m.bryant@gmail.com')
+
+        self.assertIsNone(m.project_number)
+        self.assertIsNone(m.task_number)
+        self.assertEqual(m.platform, 'NovaSeq S4')
+        self.assertEqual(m.run_type, 'PE150')
+        self.assertEqual(m.custom_primer,
+                         'No-Standard Illumina Primers are fine')
```

Comment:
No-Standard or Non-Standard?

Reply:
Should be `No-Standard...` as in `No, Standard Illumina ....`. It's just some odd formatting on their end.